server/routes/v1/post.ts

```diff
@@ -122,7 +122,7 @@ postRouter.put('/:postid/upvote', async (req: Request, res: Response) => {
       getAuthUser(res).id,
       req.params.postid
     );
-    res.status(result.status);
+    res.status(result.status).json();
```

Comment:
Why json? Isn't this a 204? For the error case we can return the json and status code. same for the other ones

Reply:
Agreed. This shouldn't be needed.

Reply:
the reason for this was mentioned in my commit comment (hit the triple dots)

the website was blocking or stalling on any interaction because despite the 204 status, they would still be waiting for a response, therefore, sending empty .json() resolves this issue